sdk/cosmos/azure-cosmos/setup.py

```diff
@@ -25,14 +25,26 @@
     version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
                         fd.read(), re.MULTILINE).group(1)
 
+if not version:
+    raise RuntimeError("Cannot find version information")
+
 with open("README.md", encoding="utf-8") as f:
     readme = f.read()
 with open("CHANGELOG.md", encoding="utf-8") as f:
     changelog = f.read()
 
+exclude_packages = [
+    "tests",
+    "tests.*",
```

Comment:
What did we need this one for?

Reply:
good catch, it's also the wrong name for cosmos - we have tests as `test` for whatever reason, ill update